IndicoV2.Abstractions/Jobs/IJobsClient.cs

```diff
@@ -12,25 +11,25 @@ public interface IJobsClient
         /// <summary>
         /// Generates <seealso cref="ISubmission"/>'s result
         /// </summary>
-        /// <param name="submissionId"></param>
-        /// <param name="cancellationToken"></param>
+        /// <param name="submissionId"><see cref="ISubmission"/>'s Id</param>
+        /// <param name="cancellationToken"><c><see cref="CancellationToken"/></c> for handling cancellation of asyncronous operations.</param>
```

Comment:
Sorry to bother you with such a small thing - I believe it should be `asynchronous`. ;)